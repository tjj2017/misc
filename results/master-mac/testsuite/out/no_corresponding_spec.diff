--- expected
+++ output
@@ -4,3 +4,3 @@
 N_Pragma (Node_Id=2444) (analyzed)
-Sloc = 8446  no_corresponding_spec.adb:8:11
+Sloc = 12542  no_corresponding_spec.adb:8:11
 Pragma_Argument_Associations = List (List_Id=-99999945)
