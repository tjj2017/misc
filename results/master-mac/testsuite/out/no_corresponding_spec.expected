Standard_Output from gnat2goto no_corresponding_spec:
----------At: Process_Pragma_Declaration----------
----------Unsupported pragma: Global----------
N_Pragma (Node_Id=2444) (analyzed)
 Sloc = 8446  no_corresponding_spec.adb:8:11
 Pragma_Argument_Associations = List (List_Id=-99999945)
 Corresponding_Aspect = N_Aspect_Specification (Node_Id=2279)
 Pragma_Identifier = N_Identifier "global" (Node_Id=2443)
 From_Aspect_Specification = True
 Is_Analyzed_Pragma = True

[assertion.1] file no_corresponding_spec.adb line 15 Ada Check assertion: SUCCESS
VERIFICATION SUCCESSFUL
